lib/ecto/adapters/sql.ex

```diff
@@ -187,7 +187,9 @@ defmodule Ecto.Adapters.SQL do
 
       @doc false
       def execute_ddl(meta, definition, opts) do
-        Ecto.Adapters.SQL.execute_ddl(meta, @conn.execute_ddl(definition), opts)
+        meta
+        |> Ecto.Adapters.SQL.execute_ddl(@conn.execute_ddl(definition), opts)
+        |> Enum.flat_map(&@conn.ddl_logs/1)
```

Comment:
It is best to avoid doing the work here and instead do it inside `Ecto.Adapters.SQL`. So I propose to call something like `Ecto.Adapters.SQL.execute_ddl(meta, @conn, definition, opts)` and do all of the work there.